test/test_module_autogeneration.py

```diff
@@ -48,6 +48,7 @@
 from alignak_test import *
 
 
+@unittest.skipIf(True, "if you want namedpipe then define it in your config !")
```

Comment:
it's linked with config.hack_old_nagios_parameters_for_arbiter. 
I think it's not used